pkg/transparentproxy/kubernetes/kubernetes_test.go

```diff
@@ -195,7 +195,7 @@ var _ = Describe("kubernetes", func() {
 						metadata.KumaTransparentProxyingInboundPortAnnotationV6:  "25206",
 						metadata.KumaSidecarUID:                                  "12345",
 						metadata.KumaTransparentProxyingEbpf:                     metadata.AnnotationEnabled,
-						metadata.KumaTransparentProxyingEbpfInstanceIPEnvVarName: "FOO_BAR_BAZ",
+						metadata.KumaTransparentProxyingEbpfInstanceIPEnvVarName: "FOO_BAR",
```

Comment:
nit: IP in value would be more realistic example

Reply:
it's not IP, but the name of env var which we are setting for kubernetes to pass us the IP address